package.json

```diff
@@ -14,7 +14,19 @@
     "storybook:prod": "build-storybook -c .storybook -o .out"
   },
   "jest": {
-    "setupTestFrameworkScriptFile": "./setupTest.js"
+    "setupFilesAfterEnv": [
+      "<rootDir>/setupTest.js"
+    ]
+  },
+  "babel": {
```

Comment:
What's the advantage of moving the babel config here? I've encountered issues with babel configs in package.json before, so it's good to know.

Reply:
It had troubles early in the process to find the presets. This fixed it for now. Since the library is EOL I thought we can move on like this and never touch it again